src/fileutils.cpp

```diff
@@ -57,3 +57,19 @@ std::vector<std::string> listDirectory(std::string sDirectory) {
    sort(s.begin(), s.end()); // sort elements
    return s;
 }
+
+// Returns a vector containing the names of the files having extension "ext" in
+// the specified directory
+std::vector<std::string> listDirectoryExt(std::string sDirectory, std::string ext) {
```

Comment:
I cannot make sDirectory const, as it is modified in the listDirectory function. Other than this, I did the modifications.